framework/src/utils/Conversion.C

```diff
@@ -446,6 +447,33 @@ stringify(const SolveType & t)
   return "";
 }
 
+std::string
+stringify(const EigenSolveType & t)
+{
+  switch (t)
+  {
+    case EST_POWER:
+      return "Power";
+    case EST_ARNOLDI:
+      return "ARNOLDI";
+    case EST_KRYLOVSCHUR:
+      return "KRYLOVSCHUR";
+    case EST_JACOBI_DAVIDSON:
+      return "Jacobi Davidson";
+    case EST_NONLINEAR_POWER:
+      return "Nonlinear Power";
+    case EST_PJFNKMO:
+      return "PJFNK with matrix only";
+    case EST_NEWTON:
+      return "Newton";
+    case EST_JFNK:
+      return "JFNK";
+    case EST_PJFNK:
+      return "Preconditioned JFNK";
```

Comment:
Seems like we should be consistent about casing here. Some are all caps while others are not

Reply:
I see. Will change that